### PR TITLE
[Merged by Bors] - feat: `exp (t * x) ≤ cosh x + t * sinh x`

### PR DESCRIPTION
**Pull request by YaelDillies**
From LeanAPAP

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/Convex/SpecificFunctions/Basic.lean [hunk 1]
```diff
@@ -216,3 +216,16 @@ theorem strictConcaveOn_log_Iio : StrictConcaveOn ℝ (Iio 0) log := by
     _ = _ := by rw [log_neg_eq_log]
 
 #align strict_concave_on_log_Iio strictConcaveOn_log_Iio
+
+namespace Real
+
+lemma exp_le_cosh_add_mul_sinh {t : ℝ} (ht : |t| ≤ 1) (x : ℝ) :
```
Comment (robertylewis) [inline Mathlib/Analysis/Convex/SpecificFunctions/Basic.lean:222]:
```suggestion
lemma exp_mul_le_cosh_add_mul_sinh {t : ℝ} (ht : |t| ≤ 1) (x : ℝ) :
```
I think?

**Review by robertylewis — COMMENTED** [review]
bors d+

File: Mathlib/Analysis/Convex/SpecificFunctions/Basic.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Analysis/Convex/SpecificFunctions/Basic.lean:222]: Sure yeah that makes sense. I was hoping the RHS would be enough to disambiguate, but oh well.

**Comment by YaelDillies** [comment]
bors merge

**Pull request closed without merging**